### DISCUSSION TITLE
blocked account

### DISCUSSION THREAD

**Discussion by Clara-Lopez** [community/community#138994] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

What is the reason why when I try to enter a sale the following occurs?
Access to your account has been suspended due to a violation of our
Since I don't break any rules, how do I get my account to stop being suspended?

**Comment by UGYnpU2nuB8QCFqDrsWZgmxMBp43WQ** [comment]
Me too.

**Comment by thanhdat2508** [comment]
Hi Clara-Lopez
I think you should to read carefully the policy or you can contact to the admin of that shop !

**Comment by ebndev** [comment]
Hi @Clara-Lopez,

The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
